examples/gallery/images/track_sampling.py

```diff
@@ -25,18 +24,40 @@
 track = pygmt.grdtrack(points=points, grid=grid, newcolname="bathymetry")
 
 fig = pygmt.Figure()
-# Plot the earth relief grid on Cylindrical Stereographic projection, masking
-# land areas
+# Create global map using a Cylindrical Stereographic projection with a
+# width of 15 centimeters
 fig.basemap(region="g", projection="Cyl_stere/150/-20/15c", frame=True)
-fig.grdimage(grid=grid, cmap="gray")
-fig.coast(land="#666666")
-# Plot the sampled bathymetry points using circles (c) of 0.15 cm size
-# Points are colored using elevation values (normalized for visual purposes)
+
+# Set up a colormap for the elevation values of the Earth relief grid
+pygmt.makecpt(cmap="gray", series=[int(grid.min()), int(grid.max()), 10])
+# Plot the Earth relief grid with color-coding for the elevation
+fig.grdimage(grid=grid, cmap=True)
+# Add a colorbar for the elevation
+fig.colorbar(
+    position="JBC+o0c/1.2c+ml",  # Place colorbar at position Bottom Center
+    frame=["af", "x+lelevation", "y+lm"],
+)
+
+# Mask the land areas in gray and plot the shorelines in black with a
+# thickness of 0.5 points
+fig.coast(land="#666666", shorelines="1/0.5p,black")
```

Comment:
Adding the shorelines makes the map look a bit more complicated, maybe it's better to remove them?

Reply:
I added the shorelines, to emphasise that only the elevation of the water masses is color-coded by the gray colormap, but the land masses are colored with a fixed gray hue.